docs/cse/rules/write-outlier-rule.md

```diff
@@ -0,0 +1,109 @@
+---
+id: write-outlier-rule
+title: Write an Outlier rule
+sidebar_label: Outlier Rule
+description: Outlier rules allow you to generate a Signal when behavior by an Entity (user) is encountered that qualifies as an outlier from expected behavior.
+keywords:
+  - sumo logic
+  - cloud siem
+  - cse
+  - outlier rule
+  - ueba
+  - behavioral analytics
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+This topic has information about Outlier rules and how to create them in the CSE UI.
+
+:::tip
+If you are new to writing rules, see [About CSE Rules](/docs/cse/rules/about-cse-rules.md) for information about rule expressions and other rule options.
```

Comment:
No need for the .md since (`/docs/...`) means it's an absolute link. You can do either one - both correct:

- Relative: `[About CSE Rules](about-cse-rules.md)`...
- Absolute: `[About CSE Rules](/docs/cse/rules/about-cse-rules)`....

Reply:
That link came in as a copy-and-paste from the "Write a First Seen rule" topic.